examples/webpack.config.js

```diff
@@ -12,7 +12,7 @@ const config = {
     mode: PROD ? "production" : "development",
     entry: ['./examples/index.js'],
     output: {
-        filename: DEV ? 'bundle.js' : 'bundle.[hash].js',
+        filename: 'bundle.[hash].js',
```

Comment:
From https://webpack.js.org/migrate/5/#clean-up-configuration
![image](https://user-images.githubusercontent.com/454175/144888321-ffd17c27-3847-455a-9461-772e69e3b746.png)

but it's optional for this example code.

See also https://webpack.js.org/guides/caching/#output-filenames

Reply:
Ah, didn't know about it. Changed it, thanks!